pkg/code-generator/model/project.go

```diff
@@ -143,40 +171,47 @@ type XDSResource struct {
 	NameField    string
 	NoReferences bool
 
-	Project      *Project
+	Project      *Version
 	ProtoPackage string // eg. gloo.solo.io
 
 	Filename string // the proto file where this resource is contained
 }
 
-func LoadProjectConfig(path string) (ProjectConfig, error) {
+func LoadProjectConfig(path string) (SoloKitProject, error) {
```

Comment:
Any particular reason this isn't returning a pointer?

Reply:
nope, just following the existing pattern